airbyte-webapp/src/packages/cloud/views/settings/integrations/DbtCloudSettingsView.tsx

```diff
@@ -21,15 +22,20 @@ export const DbtCloudSettingsView: React.FC = () => {
           initialValues={{
             serviceToken: "",
           }}
-          onSubmit={({ serviceToken }) => {
+          onSubmit={({ serviceToken }, { resetForm }) => {
             setHasValidationError(false);
```

Comment:
Just noticed: we likely also want to add a `setValidationMessage("")` here before querying. At the moment the error message stays during querying still visible but turned grey suddenly.

Reply:
Good call, will do.